.env.sample

```diff
@@ -9,3 +9,6 @@ IRMASERVER_REQUESTORS='{"keyshare_server":{"auth_method":"publickey","key_file":
 
 # Debug
 IRMASERVER_VERBOSE=2
+
+# FOR DEMO SERVER - docker version requires to remove single quote
```

Comment:
I didn't get how irma requestor and docker version are related!?

Reply:
When docker loads the `.env` file to forward the variables to irma server container, ec2 docker version does not recognize the JSON variable `IRMASERVER_REQUESTORS` when using single quotes and the irma server fails to start.

Reply:
I see!!